esp/templates/program/modules/availabilitymodule/availability_form.html

```diff
@@ -54,31 +147,130 @@
-        </tr>
-        {% endfor %}
-    {% endfor %}
-    <tr>
-        <td align="center" colspan="2"><input type="submit" class="btn btn-primary" value="Submit" /></td>
-    </tr>
-</table>
-</form>
 
-<br />
+<center>
+  <div class="wrap">
+    <div class="side"></div>
+    <div class="container1">
+      <table class="noselect" id="grid" onmouseup="down=false;" style="cursor: default;">
+        <tbody>
+          <tr>
+          {% for group in groups %}
+            <td class="group">
+              <table class="group" id="block_{{ forloop.counter }}">
+                <tbody>
+                  <tr><th class="dateHeader weekday" onclick="header(event,{{ forloop.counter }});" onmouseover="down=false;">
+                    <div class="headerText">
+                      {{ group.selections.0.slot.start|date:"D" }}<br>
+                      {{ group.selections.0.slot.start|date:"d" }}<br>
+                      {{ group.selections.0.slot.start|date:"M" }}
+                    </div>
+                  </th></tr>
+                  {% for selection in group.selections %}
+                  <tr><td nowrap="" class="proposed" onmousedown="md(this);" onmouseover="mo(this);" name="{{ selection.slot.id }}">
+                      <input type="checkbox" class="hidden" name="timeslots" value="{{ selection.slot.id }}" {% if selection.checked %}checked{% endif %} {% if selection.taken %}disabled{% endif %}>
+                      {{ selection.slot.start|date:"g:i A"}}
+                  </td></tr>
+                  {% endfor %}
+                </tbody>
+              </table>
+            </td>
+            {% if forloop.counter|divisibleby:2 %}</tr><tr><td></td></tr><tr>{% endif %}
+          {% endfor %}
+          </tr>
+        </tbody>
+      </table>
+
+    </div>
+    <div class="side">
+      <span class="summary" style="background-color: #EFEFEF;">Not Available</span>
+      <span class="summary" style="background-color: #00FF00;">Available</span>
+      <span class="summary" style="background-color: #42b3f4;">Scheduled to Teach</span>
+    </div>
+
+  </div>
+  <br>
+
+  <input type="submit" class="btn btn-primary" value="Submit" />
+</center>
+
+</form>
 
 </div>
 
+<script>
+var setting = false;
+var down = false;
+
+//Records mousedown
+function md(td) {
+  down = true;
+  setting = !isSet(td);
+  mo(td);
+}
+
+//Enables hightlighting multiple cells
+function mo(td) {
+  if (!down) return;
+  if (setting) {
+    on(td);
+  } else {
+    off(td);
+  }
+}
+
+function isSet(td) {
+  return td.className == "canDo" || td.className == "teaching";
+}
+
+//Activate checkbox
+function on(td) {
+  var checkbox = $j('input[value='+td.getAttribute("name")+']')[0]
+  if (td.className != "teaching") {
+  	td.className = "canDo";
+  	checkbox.checked = true;
+  }
+}
+
+//Deactivate checkbox
+function off(td) {
+  var checkbox = $j('input[value='+td.getAttribute("name")+']')[0]
+  if (!checkbox.disabled) {
```

Comment:
Since the on() function uses td.className in its condition (line 229), the code is clearer to read if this off() function also does. Can you change the condition to 

if (td.className == "canDo") {